__tests__/installer.test.ts

```diff
@@ -13,6 +15,61 @@ import * as installer from '../src/installer';
 
 const IS_WINDOWS = process.platform === 'win32';
 
+describe('version tests', () => {
+  each(['3.1.999', '3.1.101-preview']).test(
```

Comment:
Preview versions use this format: 5.0.0-preview.3

Reply:
Fixed it